src/Files.Uwp/Views/SettingsPages/About.xaml

```diff
@@ -1,248 +1,291 @@
-                        Margin="4"
-                        Padding="8"
-                        Text="{helpers:ResourceString Name=SettingsPrivacyPolicyMarkdownBlock/Text}" />
-                </local:SettingsBlockControl.ExpandableContent>
-            </local:SettingsBlockControl>
-
-            <local:SettingsBlockControl
-                Title="{helpers:ResourceString Name=OpenLogLocation}"
-                HorizontalAlignment="Stretch"
-                ButtonCommand="{x:Bind ViewModel.OpenLogLocationCommand}"
-                IsClickable="True">
-                <local:SettingsBlockControl.Icon>
-                    <FontIcon Glyph="&#xE838;" />
-                </local:SettingsBlockControl.Icon>
-
-                <FontIcon
-                    FontSize="14"
-                    Foreground="{ThemeResource TextFillColorPrimaryBrush}"
-                    Glyph="&#xE2B4;" />
-            </local:SettingsBlockControl>
-
-            <local:SettingsBlockControl Title="{helpers:ResourceString Name=ThirdPartyLicenses}" HorizontalAlignment="Stretch">
-                <local:SettingsBlockControl.Icon>
-                    <FontIcon Glyph="&#xE99A;" />
-                </local:SettingsBlockControl.Icon>
-                <local:SettingsBlockControl.ExpandableContent>
-                    <StackPanel Padding="4" Spacing="4">
-                        <HyperlinkButton
-                            AutomationProperties.Name="QuickLook"
-                            Content="QuickLook"
-                            NavigateUri="https://github.com/QL-Win/QuickLook/blob/master/LICENSE-GPL.txt" />
-
-                        <HyperlinkButton
-                            AutomationProperties.Name="{helpers:ResourceString Name=SettingsFluentUISystemIconsLicenseButton/AutomationProperties/Name}"
-                            Content="{helpers:ResourceString Name=SettingsFluentUISystemIconsLicenseButton/Content}"
-                            NavigateUri="https://github.com/microsoft/fluentui-system-icons/blob/master/LICENSE" />
-                    </StackPanel>
-                </local:SettingsBlockControl.ExpandableContent>
-            </local:SettingsBlockControl>
-        </StackPanel>
-    </Grid>
+	x:Class="Files.Uwp.SettingsPages.About"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:helpers="using:Files.Uwp.Helpers"
+	xmlns:i="using:Microsoft.Xaml.Interactivity"
+	xmlns:ic="using:Microsoft.Xaml.Interactions.Core"
+	xmlns:local="using:Files.Uwp.UserControls.Settings"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	mc:Ignorable="d">
+	<Page.Resources>
+		<ResourceDictionary>
+			<ResourceDictionary.MergedDictionaries>
+				<ResourceDictionary Source="/ResourceDictionaries/RightAlignedToggleSwitchStyle.xaml" />
```

Comment:
I don't think that's needed really. I found out that you could just set MinWidth="0" and it'll be aligned.

Reply:
I'm keen to go through and remove these styles at some point but it's not related to this PR as this line was already there before.